src/ai/backend/manager/registry.py

```diff
@@ -155,6 +155,7 @@
     SessionNotFound,
     TooManySessionsMatched,
 )
+from .event_dispatcher.dispatch import EventLogger
 from .exceptions import MultiAgentError
 from .models import (
```

Comment:
It's a structural inversion.

Reply:
I tried to refactor this part in this branch but it will make so many changes in this entire code base. Can I fix the issue as a follow-up of this work?